Makefile

```diff
@@ -186,14 +189,43 @@ docclean:
 
 RAWVER=$(shell $(PYTHON) tools/getnodeversion.py)
 VERSION=v$(RAWVER)
+ifdef NIGHTLY
+# For nightly builds, you must set NIGHTLY to "nightly", "next-nightly" or "rc".
+# For the nightly and next-nightly case, you need to set DATESTRING and COMMIT
+# in order to properly name the build.
+# For the rc case you need to set RC to an appropriate RC number
+ifeq ($(NIGHTLY),rc)
+ifndef RC
+$(error RC is not set)
+endif
+TAG=rc$(RC)
```

Comment:
Perhaps we could make this a bit more generic as in:
```
TAG=$(TAGSUFFIX)
```

.. and then pass TAGSUFFIX=rc1 (or beta2, privatexyx, etc)?